README.md

```diff
@@ -1,77 +1,85 @@
-![](https://github.com/nomad-coe/nomad-schema-plugin-example/actions/workflows/actions.yml/badge.svg)
-![](https://coveralls.io/repos/github/FAIRmat-NFDI/nomad-analysis/badge.svg?branch=main)
```

Comment:
What is the reason for removing the coveralls badge?

Reply:
My bad, I will include it back again (same for the badge of the workflow)